### PR TITLE
[Build] Generate resources accessor for ObjC targets

### PR DESCRIPTION
**Pull request by aciidb0mb3r**
This adds support for generating resources accessor for ObjC targets.

<rdar://problem/56816817>

### CODE REVIEW

**Comment by aciidb0mb3r** [comment]
@swift-ci smoke test

File: Sources/PackageModel/Sources.swift [hunk 1]
```diff
@@ -39,6 +39,16 @@ public struct Sources {
...
         })
     }
+
+    /// Returns true if the sources contain C++ files.
+    public var containsObjcFiles: Bool {
+        return paths.contains(where: {
+            guard let ext = $0.extension else {
+                return false
+            }
+            return ext == SupportedLanguageExtension.m.rawValue
```
Comment (neonichu) [inline Sources/PackageModel/Sources.swift]: Should this also be done for ObjC++?

Comment (aciidb0mb3r) [reply Sources/PackageModel/Sources.swift]: yeah

Comment (aciidb0mb3r) [reply Sources/PackageModel/Sources.swift]: Will fix this in a followup.

**Pull request merged**